docs/release-notes/v11.4.0.md

```diff
@@ -19,6 +19,29 @@ _July 06th, 2023_
 
 ### New Features
 
+#### Single Views and CRUD Collections are now available as sources inside the No Code ER Schema
+
+Single Views and CRUD Collections can now be used as an input for the No Code ER Schema. The new sources will be available in the right panel of the canvas and can be easily added to the canvas with a simple drag-and-drop, just as it is already possible with Projections. When added to the ER Schema, Single Views and CRUD Collections are then included as available dependencies inside the No Code Aggregation.
+
+![Collections inside ER Schema](./img/overview/collections-er-schema.png)
+
+#### Index autogeneration for projections creates indexes for `__internal_counter` and `__internal_counterType`
+
+The Console can now automatically generate the `__internal_counter` and `__internal_counterType` indexes, which have been introduced from `v7.5.0` of the Real-Time Updater in the update Projection queries.
+
+#### The Aggregation config map inside the Single View Creator microservice is automatically handled
+
+When the No Code Aggregation is enabled, the Aggregation config map is automatically handled by Fast Data. The config map will be editable from the No Code Aggregation screen, without needing to access the Microservice section and the related Single View Creator. Thus, the config map will be displayed as read-only inside this section.
```

Comment:
This is not when the no code is enabled, it is done upon connecting the svc to the sv, even if the no-code is disabled. Since from that point on you can modify the config maps from the configuration tabs in the sv/svc page

Reply:
Thanks, updated